scripts/install

```diff
@@ -2,123 +2,28 @@
-fi
-if [ -z "$WITH_IPFS" -o "$WITH_IPFS" != "true" -a "$WITH_IPFS" != "false" ]; then
-    read -p "Install IPFS (Y/n)? " -n 1 -r
-    echo ""
-    if [[ $REPLY =~ ^[Nn]$ ]]; then
-        echo -e "\e[1;31mIPFS installation will be skipped\e[0m"
-        WITH_IPFS=false
-    else
-        echo -e "\e[1;32mIPFS will be installed\e[0m"
-        WITH_IPFS=true
-    fi
-fi
-
-# Get tools
-if ! [ "$(which git)" ] || ! [ "$(which nodejs)" ] || ! [ "$(which iperf3)" ]; then
-    sudo apt-get update
-    sudo apt-get install git nodejs iperf3 -y
+# Get git
+sudo apt-get update
+if ! [ "$(which git)" ]; then
+    sudo apt-get install git -y
 fi
 
-# Download prototype-cjdns-pi2 repo and checkout TAG_PROTOTYPE_CJDNS_PI2 tag
+# Download prototype-cjdns-pi2 repo
 if ! [ -d "prototype-cjdns-pi2" ]; then
     git clone https://github.com/tomeshnet/prototype-cjdns-pi2.git
-    cd prototype-cjdns-pi2 && git checkout $TAG_PROTOTYPE_CJDNS_PI2 && cd ..
-fi
-
-# Download cjdns repo and checkout TAG_CJDNS tag
-if ! [ -d "/opt/cjdns" ]; then
-    here=`pwd`
-    sudo git clone https://github.com/cjdelisle/cjdns.git /opt/cjdns
-    cd /opt/cjdns && sudo git checkout $TAG_CJDNS && cd $here
 fi
 
-# Build cjdns
-if ! [ -x "/opt/cjdns/cjdroute" ]; then
-    here=`pwd`
-    cd /opt/cjdns && sudo Seccomp_NO=1 NO_NEON=$CJDNS_NO_NEON CFLAGS="$CJDNS_CFLAGS" ./do && cd $here
+# Checkout provided tag or default to master
+if [ -z "$TAG_PROTOTYPE_CJDNS_PI2" ]; then
+    TAG_PROTOTYPE_CJDNS_PI2=master
 fi
+cd prototype-cjdns-pi2
+git checkout $TAG_PROTOTYPE_CJDNS_PI2
 
-# Install cjdns to /usr/bin
-if ! [ -x "/usr/bin/cjdroute" ]; then
-    sudo cp /opt/cjdns/cjdroute /usr/bin/cjdroute
-fi
-
-# Generate cjdns configurations
-if ! [ -f "/etc/cjdroute.conf" ]; then
-    sudo /usr/bin/cjdroute --genconf | sudo tee --append /etc/cjdroute.conf > /dev/null
-fi
-
-# Configure systemd to start cjdns.service on system boot
-sudo cp /opt/cjdns/contrib/systemd/cjdns.service /lib/systemd/system/cjdns.service
-sudo chmod 644 /lib/systemd/system/cjdns.service
-sudo cp /opt/cjdns/contrib/systemd/cjdns-resume.service /lib/systemd/system/cjdns-resume.service
-sudo chmod 644 /lib/systemd/system/cjdns-resume.service
-sudo systemctl daemon-reload
-sudo systemctl enable cjdns.service
-
-# 802.11s Mesh Point interface
-if [ ! -z "$WITH_MESH_POINT" -a "$WITH_MESH_POINT" == "true" ]; then
-    source prototype-cjdns-pi2/scripts/mesh-point/install
-fi
-
-# WiFi Access Point on RPi3
-if [[ $RPI_REVISION == *"a02082"* || $RPI_REVISION == *"a22082"* ]] && [ ! -z "$WITH_WIFI_AP" -a "$WITH_WIFI_AP" == "true" ]; then
-    source prototype-cjdns-pi2/scripts/hostapd/install
-fi
-
-# IPFS
-if [ ! -x "$(command -v ipfs)" ] && [ ! -z "$WITH_IPFS" -a "$WITH_IPFS" == "true" ]; then
-    source prototype-cjdns-pi2/scripts/ipfs/install
-fi
+# Export environment variables
+export WITH_MESH_POINT
+export WITH_WIFI_AP
+export WITH_IPFS
 
-# Reboot device
-sudo reboot
+# Run the actual installation script
+./install2
```

Comment:
`./install2` is in `scripts`. https://github.com/tomeshnet/prototype-cjdns-pi2/blob/things/scripts/install#L20 should probably `cd` into it?